txpool/pool.go

```diff
@@ -624,8 +625,18 @@ func (p *TxPool) best(n uint16, txs *types.TxsRlp, tx kv.Tx, onTopOf, availableG
 
 			mt := best.ms[i]
 
-			if isYielding && mt.alreadyYielded {
-				continue
+			if len(toSkip) > 0 {
+				skip := false
+				for _, id := range toSkip {
+					if mt.Tx.IDHash == id {
+						skip = true
+						break
+					}
+				}
+				if skip {
+					fmt.Printf("skipping yielded")
```

Comment:
Let's remove the debug printf

Reply:
Absolutely, can't believe i missed that when reviewing myself